tests/e2e/performance/csi_tests/test_pod_reattachtime.py

```diff
@@ -121,13 +121,18 @@ def test_pod_reattach_time_performance(
         node_one = worker_nodes_list[0]
         node_two = worker_nodes_list[1]
 
-        time_measures, files_written_list, data_written_list = ([], [], [])
+        time_measures, csi_time_measures, files_written_list, data_written_list = (
+            [],
+            [],
+            [],
+            [],
+        )
         for sample_index in range(1, samples_num + 1):
             # Create a PVC
             accessmode = constants.ACCESS_MODE_RWX
             if self.interface == constants.CEPHBLOCKPOOL:
                 accessmode = constants.ACCESS_MODE_RWO
-
+            csi_start_time = self.get_time("csi")
```

Comment:
Not sure if I got this, Is this where you get the start time to calculate node-reattach time?

Reply:
This is to take the timestamp starting from which we later parse the logs
for pod reattach time